### PR TITLE
Escape tag values that ends with backslash

### PR DESCRIPTION
**Pull request by vaniakov**
If we try to write tag value that ends with backslash we get an InfluxDB error:
```bash
$ ipython3
Python 3.5.2 (default, Nov 17 2016, 17:05:23)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.2.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from influxdb import InfluxDBClient
   ...: client = InfluxDBClient()
   ...: client.create_database('statistics')
   ...: client.switch_database('statistics')
   ...:

In [2]: data = [
   ...:     {
   ...:      'measurement': 'disk_usage',
   ...:      'tags':{
   ...:         'device': 'C:\\'
   ...:         },
   ...:         'fields': {
   ...:             'total_bytes': 9846712221,
   ...:             'free_bytes': 971245124
   ...:         }
   ...:     }
   ...: ]

In [3]: client.write_points(data)
---------------------------------------------------------------------------
InfluxDBClientError                       Traceback (most recent call last)
<ipython-input-3-66de51fff414> in <module>()
----> 1 client.write_points(data)

/usr/local/lib/python3.5/dist-packages/influxdb/client.py in write_points(self, points, time_precision, database, retention_policy, tags, batch_size, protocol)
    454                                       database=database,
    455                                       retention_policy=retention_policy,
--> 456                                       tags=tags, protocol=protocol)
    457
    458     def _batches(self, iterable, size):

/usr/local/lib/python3.5/dist-packages/influxdb/client.py in _write_points(self, points, time_precision, database, retention_policy, tags, protocol)
    504                 params=params,
    505                 expected_response_code=204,
--> 506                 protocol=protocol
    507             )
    508

/usr/local/lib/python3.5/dist-packages/influxdb/client.py in write(self, data, params, expected_response_code, protocol)
    300             data=data,
    301             expected_response_code=expected_response_code,
--> 302             headers=headers
    303         )
    304         return True

/usr/local/lib/python3.5/dist-packages/influxdb/client.py in request(self, url, method, params, data, expected_response_code, headers)
    261             return response
    262         else:
--> 263             raise InfluxDBClientError(response.content, response.status_code)
    264
    265     def write(self, data, params=None, expected_response_code=204,

InfluxDBClientError: 400: {"error":"unable to parse 'disk_usage,device=C:\\\\ free_bytes=971245124i,total_bytes=9846712221i': invalid tag format"}
```

This is workaround for InfluxDB [issue](https://github.com/influxdata/influxdb/issues/5231), the same reproduces in influx cli:

```bash
$ influx
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version 0.10.0
InfluxDB shell 0.10.0
> use statistics
Using database statistics
> insert disk_usage,device=C:\\\\ free_bytes=971245124i,total_bytes=9846712221i
ERR: {"error":"unable to parse 'disk_usage,device=C:\\\\\\\\ free_bytes=971245124i,total_bytes=9846712221i': invalid tag format"}

> insert disk_usage,device=C:\\  free_bytes=971245124i,total_bytes=9846712221i
> select * from disk_usage
name: disk_usage
----------------
time                    device  free_bytes      total_bytes
1510913194937625701     C:\     971245124       9846712221
```

### CODE REVIEW

**Comment by xginn8** [comment]
Thanks for contributing! Would you mind adding a test case for this PR?

**Comment by vaniakov** [comment]
@xginn8 sure.

File: influxdb/tests/test_line_protocol.py [hunk 1]
```diff
@@ -39,9 +40,10 @@ def test_make_lines(self):
             ]
         }
 
+        print(line_protocol.make_lines(data))
```
Comment (aviau) [inline influxdb/tests/test_line_protocol.py]: Remove the print please

**Comment by vaniakov** [comment]
@xginn8, @aviau could you please verify changes?

**Comment by vaniakov** [comment]
Thank you!

**Pull request merged**